project_euler/problem_074/sol2.py

```diff
@@ -16,9 +16,13 @@
     the chains of non repeating items.
     The generation of the chain stops before a repeating item
     or if the size of the chain is greater then the desired one.
-    After generating each chain, the length is checked and the counter increases.
+    After generating each chain, the length is checked and the
+    counter increases.
 """
 
+FACTORIAL_CACHE = {}
+FACTORIAL_SUM_CACHE = {}
```

Comment:
This is not a constant value, please convert the variable names to lowercase and apply the changes wherever necessary.

Reply:
It's similar in `sol1.py` for chain length dict, BTW